alvr/xtask/flatpak/README.md

```diff
@@ -21,20 +19,62 @@ git clone https://github.com/alvr-org/ALVR.git
 cd ALVR
 ```
 
-3. Build and install the flatpak
+3. Build and install the flatpak via the provided script
 
 ```
-flatpak run org.flatpak.Builder --user --install --force-clean .flatpak-build-dir alvr/xtask/flatpak/com.valvesoftware.Steam.Utility.alvr.json
+cd alvr/xtask/flatpak
+./build_and_install.sh  
```

Comment:
this script requires host flatpak builder, but before it was from flatpak itself

Reply:
`flatpak-builder` vs `flatpak run org.flatpak.Builder`

Reply:
Besides this, everything else works fine on amdgpu
If this changed it can be merged just fine